test/functional/test_framework/util.py

```diff
@@ -87,6 +87,30 @@ def assert_raises_process_error(returncode, output, fun, *args, **kwds):
     else:
         raise AssertionError("No exception raised")
 
+def assert_raises_process_error(returncode, output, fun, *args, **kwds):
```

Comment:
This was already backported, see code above

Reply:
https://github.com/dashpay/dash/pull/3106/commits/bdc0ffec3da9897120fa661835ff9e5f4f23f38a